### ISSUE TITLE
Descriptive information is not announced by Screen reader for the 'icon' controls under 'Open success snackbar' table 

### ISSUE THREAD

**Issue by msftedad** [ag-grid/ag-grid#6972]
**Steps to Reproduce**

1. Open URL: https://y8qw2z.csb.app/
2. Navigate to Icon column using keyboard.
3. Turn on the Screen Readers
4.  Verify for screen reader announcement all the mentioned controls in the grid.

**Current behavior**
Descriptive information is not announced by Screen reader for the 'icon' controls under 'Open success snackbar' table.

Observations:

Narrator: View icon column 3 read only
NVDA: View icon column 3
JAWS: View icon column 3

**Expected behavior**
Screen reader should announce descriptive information for the 'icon' controls under 'Open success snackbar' table tab should not announce unnecessary information as read-only.

EX: Screen Readers should announce as " Time Icon row 2 column 3"

**Please tell us about your environment:**
OS: Windows 11 Enterprise Build: 22621.2134

* **AG Grid version:** X.X.X
29.3.3

* **Browser:**
Chrome version: 116.0.5845.96 (Official Build) (64-bit)

* **Language:** [all | TypeScript X.X | ES6/7 | ES5]
 ES6

https://github.com/ag-grid/ag-grid/assets/93735775/c088c376-12d2-45c7-b22c-415947c42274

**Comment by zoheil-ag** [comment]
Hi @msftedad,

Thanks for raising this.

We've had a look at the sandbox sample you provided and "View Details" is the tooltip upon hover. However this tooltip doesn't get focused and screen readers would only read things that are focused.

So the best way for you to get this announced is to add the aria-label to the div of the cellRenderer.

In your code if we add `<div aria-label="View Details">` in the div of your tooltip the screen reader announces it. Please give this a try.

Thank you,
Zoheil
